reference/commands/output/install.rst

```diff
@@ -50,69 +50,123 @@ The output JSON contains a two first level keys:
-                           "version": "1.2.11",
-                           "description": "A Massively Spiffy Yet Delicately Unobtrusive Compression Library (Also Free, Not to Mention Unencumbered by Patents)"
-                       }
-                   }
-               ]
-           }
-       ]
-   }
-
+    {
+        "error":false,
+        "installed":[
+            {
+                "recipe":{
+                    "id":"OpenSSL/1.0.2l@conan/stable",
+                    "downloaded":true,
+                    "exported":false,
+                    "error":null,
+                    "remote":"https://api.bintray.com/conan/conan/conan-center",
+                    "time":"2018-11-29T11:59:53.601813",
+                    "dependency":true,
+                    "name":"OpenSSL",
+                    "version":"1.0.2l",
+                    "user":"conan",
+                    "channel":"stable"
+                },
+                "packages":[
+                    {
+                        "id":"606fdb601e335c2001bdf31d478826b644747077",
+                        "downloaded":true,
+                        "exported":false,
+                        "error":null,
+                        "remote":"https://api.bintray.com/conan/conan/conan-center",
+                        "time":"2018-11-29T12:00:03.874284",
+                        "built":false,
+                        "cpp_info":{
+                            "includedirs":[
+                                "include"
+                            ],
+                            "libdirs":[
+                                "lib"
+                            ],
+                            "resdirs":[
+                                "res"
+                            ],
+                            "bindirs":[
+                                "bin"
+                            ],
+                            "builddirs":[
+                                ""
+                            ],
+                            "libs":[
+                                "ssleay32",
+                                "libeay32",
+                                "crypt32",
+                                "msi",
+                                "ws2_32"
+                            ],
+                            "rootpath":"C:/Users/user/.conan/data/OpenSSL/1.0.2l/conan/stable/package/606fdb601e335c2001bdf31d478826b644747077",
+                            "version":"1.0.2l",
+                            "description":"OpenSSL is an open source project that provides a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols",
+                            "public_deps":[
+                                "zlib"
+                            ]
+                        }
+                    }
+                ]
+            },
+            {
```

Comment:
I will substitute the second element with `...`, is there anything interesting in it?

Reply:
It is always good to have a second example to see if anything changes in respect to the other (although it is not the case)

Reply:
These are the docs, not testing. I would add all the expected fields (with cardinality if needed) to only one example. We already have ~400 pages in docs, we don't need more 😆

Reply:
These are the docs, not testing. I would add all the expected fields (with cardinality if needed) to only one example. We already have ~400 pages in docs, we don't need more